cypress/integration/prospects/profile.js

```diff
@@ -8,6 +8,30 @@ describe('Prospects profile info', () => {
       // open the prospect profile
       cy.get('div[id="prospect-' + index + '"').click()
 
+      // open the profile menu and check if it is visible
+      cy.get('#profileAction').click()
+      cy.get('#actions').should('be.visible')
+      
+      // clicks the first and only item in the main menu,
+      // opens the modal window and confirms the action
+      cy.get('ul li').should('have.length', 1)
+      cy.get('#actions').find('li').first().click()
```

Comment:
in order to have tests that are easier to read, we should have an id like delete-button so it's clear what happens in the test